lib/cryptoexchange/exchanges/bitstamp/services/pairs.rb

```diff
@@ -0,0 +1,26 @@
+module Cryptoexchange::Exchanges
+  module Bitstamp
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        MARKET = Bitstamp::Market
```

Comment:
@dominicwong617 open to suggestion here.

This is used to determine whether to grab pairs via a URL, or fallback to YML

If the `market_url` is defined, it will take the URL. Otherwise it will try to read a YML.
That `MARKET` name is used to find the `yml` filename. If we have a better way to propagate the exchange name to the superclass, that would be better